### PR TITLE
Test 404 not found global

### PR DESCRIPTION
**Pull request by asanzgom**
Test to check the 404 not found global page

image

### CODE REVIEW

**Review by riccardo-forina — APPROVED** [review]
I don't think we really need to test this on an e2e test, but since it's cheap enough we can add it.

File: tests/kas.spec.ts [hunk 1]
```diff
@@ -120,3 +120,9 @@ test('test fail to create Kafka instance with the same name', async ({ page }) =
 
   await deleteKafkaInstance(page, testInstanceName);
 });
+
+// test_4kafka.py test_kafka_404_not_found_global
+test('404 not found Global', async ({ page }) => {
```
Comment (andreaTP) [inline tests/kas.spec.ts:125]: this test doesn't require the setup we are doing in this file, it should go in `tests/auth.spec.ts` and possibly rename the file to something more generic.

**Review by andreaTP — CHANGES_REQUESTED** [review]
I don't think as well that we should test at the E2E level that React Router works as expected, but, well, is cheap enough ... so ok ...

One note though on where to locate this test.

File: tests/kas.spec.ts (hunk 1 above)
Comment (asanzgom) [reply tests/kas.spec.ts:125]: ok, what new name do you suggest for `auth.spce.ts` ?

Comment (andreaTP) [reply tests/kas.spec.ts:125]: open to alternatives ... `basic.spec.ts` maybe? `generic.spec.ts` / `misc.spec.ts` ? not sure, naming is hard ...

Comment (asanzgom) [reply tests/kas.spec.ts:125]: ok so `basic.spec.ts` moving and we can merge and spend no more time on it

**Pull request merged**